app/queries/events_query.rb

```diff
@@ -0,0 +1,39 @@
+class EventsQuery
```

Comment:
Why is this in a `queries` subfolder?

Reply:
Mostly trying to follow query object pattern here. 

https://medium.flatstack.com/query-object-in-ruby-on-rails-56ea434365f0

Both the controller and the model seem very crowded and found it better here.

Reply:
Ok, but the example is about ActiveRecord queries not Elasticsearch. `relation` has a very different meaning there.